src/h5web/App.tsx

```diff
@@ -42,21 +43,25 @@ function App() {
           onChangeInspecting={setInspecting}
           onSelectPath={setSelectedPath}
         />
-        <ErrorBoundary
-          resetKeys={[selectedPath, isInspecting]}
-          FallbackComponent={ErrorFallback}
-        >
-          <Suspense fallback={<LoadingFallback isInspecting={isInspecting} />}>
-            {isInspecting ? (
-              <MetadataViewer
-                path={selectedPath}
-                onSelectPath={setSelectedPath}
-              />
-            ) : (
-              <VisPackChooser path={selectedPath} />
-            )}
-          </Suspense>
-        </ErrorBoundary>
+        <ConfigProvider>
+          <ErrorBoundary
+            resetKeys={[selectedPath, isInspecting]}
+            FallbackComponent={ErrorFallback}
+          >
+            <Suspense
+              fallback={<LoadingFallback isInspecting={isInspecting} />}
+            >
+              {isInspecting ? (
+                <MetadataViewer
+                  path={selectedPath}
+                  onSelectPath={setSelectedPath}
+                />
+              ) : (
+                <VisPackChooser path={selectedPath} />
+              )}
+            </Suspense>
+          </ErrorBoundary>
+        </ConfigProvider>
```

Comment:
I decided to add it here rather than to expose it in `@h5web/app`.

Reply:
I would rename it `VisConfigProvider`. `ConfigProvider` is awfully generic